OMCompiler/Compiler/Script/CevalScriptBackend.mo

```diff
@@ -3662,7 +3675,7 @@ algorithm
         end if;
         cmd := "docker run "+(if uid<>0 then "--user " + String(uid) else "")+" --rm -w /fmu -v "+volumeID+":/fmu "+stringDelimitList(rest," ")+ " sh -c " + dquote +
                "cd " + dquote + "/fmu/" + System.basename(fmutmp) + "/sources" + dquote + " && " +
-               "./configure --host="+quote+host+quote+" CFLAGS="+quote+"-Os"+quote+" CPPFLAGS=-I/fmu/fmiInclude LDFLAGS= && " +
+               "./configure --host="+quote+host+quote+" CFLAGS="+quote+"-Os"+quote+" CPPFLAGS="+quote+"-I. -I/fmu/fmiInclude"+quote+" LDFLAGS= && " +
```

Comment:
@sjoelund Do you know what is missing for the Docker cross-build-fmu test?
[Jenkins output](https://test.openmodelica.org/jenkins/blue/organizations/jenkins/OpenModelica/detail/PR-897/27/pipeline/237#step-733-log-67) suggests that `CPPFLAGS` is wrong or not set.

I added a new header file that needs `-I.` for the compiler to be set and simply added it to the other Makefiles and this docker command.

Reply:
Nevermind, I found a solution.